woocommerce/woocommerce
# Entrepreneur Signup: Show WordPress logo when entering Design With AI from Entrepreneur signup flow.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Show WordPress logo when entering Design With AI from Enterpreneur signup flow.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Sync to your WoA site on Entrepreneur plan.
2. Go to Design With AI `your_site_here.wpcomstaging.com/wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store%2Fdesign-with-ai&ref=entrepreneur-signup`.
3. You should see a black & white WordPress logo on Design With AI flow.
4. You should see the Blue logo now replaced with black & white WordPress logo on Site Assembler flow.
5. Ignore the mispositioning of Site Title, that is addressed in https://github.com/woocommerce/woocommerce/pull/47581.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>

<img width="1327" alt="2024-05-17_19-59-10" src="https://github.com/woocommerce/woocommerce/assets/1287077/240f5f8a-f100-4b7d-b0e6-aef2f9b7b3c7">

<img width="1327" alt="2024-05-17_19-59-18" src="https://github.com/woocommerce/woocommerce/assets/1287077/2e1d0217-fa4b-41bb-9089-75cf88efe315">

<img width="1320" alt="2024-05-17_19-58-08" src="https://github.com/woocommerce/woocommerce/assets/1287077/f626d38e-f3ca-4f33-81eb-172f684af4f8">

<img width="1333" alt="2024-05-17_21-13-34" src="https://github.com/woocommerce/woocommerce/assets/1287077/c6e0f24e-2512-4012-91ab-5648a782732a">

<img width="1336" alt="2024-05-17_21-18-19" src="https://github.com/woocommerce/woocommerce/assets/1287077/7ff9b370-186d-49af-bc0a-884997d298aa">
